cmd/promtool/main.go

```diff
@@ -641,3 +659,185 @@ func (j *jsonPrinter) printLabelValues(v model.LabelValues) {
 	//nolint:errcheck
 	json.NewEncoder(os.Stdout).Encode(v)
 }
+
+// ObfuscateIndexFile renames the symbols of the given index file
+// and re-writes the index file at given output path.
+func ObfuscateIndexFile(inputFilePath, outputFilePath string) int {
```

Comment:
Can we return `error` and not invent yet another way of returning errors? (:  Also can we put this in separate package, so we can use it in Thanos/Cortex/XYZ at some point programmatically? (:

Reply:
Sounds fine. I can do that.